From c2bc880b4cd659c97d7cd30220cf940d17497eca Mon Sep 17 00:00:00 2001
From: Uma Shankar <uma.shankar@intel.com>
Date: Wed, 31 May 2017 15:40:54 +0530
Subject: [PATCH 3384/5242] drm: Add HDR capabilty field to plane structure

commit  9c7802fa3e57ec3b39c9036c8c6674d3197f088f from
https://source.codeaurora.org/external/imx/linux-imx.git

Hardware may have HDR capability on certain plane
engines. Enabling the same in drm plane structure
so that this can be communicated to user space.

Each drm driver should set this flag to true for planes
which support HDR.

Signed-off-by: Uma Shankar <uma.shankar@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/drm/drm_plane.h |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/drm/drm_plane.h b/include/drm/drm_plane.h
index 26fa50c..aa47b12 100644
--- a/include/drm/drm_plane.h
+++ b/include/drm/drm_plane.h
@@ -560,6 +560,9 @@ struct drm_plane {
 
 	enum drm_plane_type type;
 
+	/* Value of true:1 means HDR is supported */
+	bool hdr_supported;
+
 	/**
 	 * @index: Position inside the mode_config.list, can be used as an array
 	 * index. It is invariant over the lifetime of the plane.
-- 
1.7.9.5

